apps/mobile/src/screens/auth/auth.tsx

```diff
@@ -13,11 +18,20 @@ import {
 	useCallback,
 	useEffect,
 } from "~/libs/hooks/hooks";
-import { type UserSignUpRequestDto } from "~/packages/users/users";
+import { globalStyles } from "~/libs/styles/styles";
+import {
+	type UserSignInRequestDto,
+	type UserSignUpRequestDto,
+} from "~/packages/users/users";
 import { actions as authActions } from "~/slices/auth/auth";
 import { actions as userActions } from "~/slices/users/users";
 
 import { SignInForm, SignUpForm } from "./components/components";
+import { styles } from "./styles";
+
+const IOS_KEYBOARD_OFFSET = 40;
+const ANDROID_KEYBOARD_OFFSET = 0;
+const isIos = Platform.OS === "ios";
```

Comment:
Let's create these two helpers and move them to ~libs/helpers to be able to use them within the other components
```
const isIos = () => Platform.OS === 'ios';
const isAndroid = () => Platform.OS === 'android';
```

Reply:
Add this to helpers, but use two different due to linter error - unused export
  `<KeyboardAvoidingView
        behavior={isAndroid() ? "height" : "padding"}
	keyboardVerticalOffset={
		isIos() ? IOS_KEYBOARD_OFFSET : ANDROID_KEYBOARD_OFFSET
	}
>`